scripts/windows/run.py

```diff
@@ -0,0 +1,6 @@
+import sys
+
+from papis.commands.default import run
+
+
+sys.exit(run())
```

Comment:
This file doesn't seem used?

Reply:
Yes, in the spec:

 https://github.com/papis/papis/pull/795/files#diff-f61ac8d297e1b0683b7e844d79087c3920b6a9d6bf5d690679d3756372af32d1R12